src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), padding and margin values should be written in a shorthand form if they generate a repeated pattern (e.g., `32px 16px` should be written as `32px 16px`).